### PR TITLE
feat(web): Ajout la page et le popup de confirmation de mail

### LINKED ISSUE #125: [1JH] UI de la page et la popup de confirmation de mail
## Describe the issue
C'est une interface qui montre la page avec un titre 'please check your mail', le but est pour entrer le code que Voron envoie dans la boîte de mail d'utilisateur

## Resource request
Utiliser la maquette de Figma

## Definition of done
- [x] Réalisation du Popup 'please check your email'
- [x] Réalisation de la Page 'Succeed to confirm your email'
- [x] Connecter la page de la confirmation à la page login
- [x] Respecter le style de figma
- [x] Ajouter une alert pour donner une réaction que la demande a été envoyé si y'a du temps
- [x] Intégration API confirmation email à la page
- [x] Intégration API resend email au Popup

## testing plan
/ 

## JEH
1

### PR DESCRIPTION
**Pull request by claraxuxu**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Fixes

Specify which issue this PR targets, like so:

- fixes #125 

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [ ] I have added the needed labels
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

**Comment by claraxuxu** [comment]
Quand on clique le lien de confirmation, il arrive pas à reconnaître l'url afin de réorienter sur la page de confirmation email à cause de 'https' au local, jsp si ça va changer qqch en prod...

File: web/src/pages/SignUp/ConfirmEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    const confirmFun = async () => {
+        try {
+            await axios
+                .post(`${config.apiUrl}/confirm?token=${token}`, {
+                    password: pwd,
+                })
+                .then(() => {
+                    setConfirmed(true);
+                    console.log('ICI');
```
Comment (djnnvx) [inline web/src/pages/SignUp/ConfirmEmail.tsx]: maybe u can delete this

File: web/src/pages/SignUp/SignUp.tsx [hunk 2]
```diff
@@ -88,6 +88,7 @@ export default function SignUp() {
         const { username, email, role, password } = state;
         if (email !== '' && password.length > 7 && username.length > 4) {
             try {
+                console.log('here?');
```
Comment (djnnvx) [inline web/src/pages/SignUp/SignUp.tsx]: u can delete this too

**Review by djnnvx — APPROVED** [review]
looks good bro) well done

**Pull request merged**